charts/posthog/README.md

```diff
@@ -1,34 +1,37 @@
-# PostHog
+# PostHog Helm Chart
 
-🦔 [PostHog](https://posthog.com/) is developer-friendly, open-source product analytics.
-
-## TL;DR;
+> :warning: **This chart is not ready yet**: Be careful!
 
-```console
-$ cd chart
-$ helm install --timeout 20m posthog .
-```
+🦔 [PostHog](https://posthog.com/) is developer-friendly, open-source product analytics.
 
 ## Introduction
 
 This chart bootstraps a [PostHog](https://posthog.com/) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
 
-This installation is intended for larger-scale usage (> 1M events/month) an
+This installation is geared towards larger scalability (> 1M events per month).
```

Comment:
I thought the plan was everyone uses this, so I'd just drop this line completely?

Reply:
Please add a note to the EPIC ticket or drop the PR you'd like. I like it here since it helps justify why this exists vs the main chart.